### ISSUE TITLE
Error with Inversify.JS

### ISSUE THREAD

**Issue by iliyaZelenko** [nuxt/nuxt.js#6076]
### Version

[v2.8.1](https://github.com/nuxt.js/releases/tag/v2.8.1)

### Reproduction link

[https://codesandbox.io/s/nuxtnuxtjs-typescript-pub29](https://codesandbox.io/s/nuxtnuxtjs-typescript-pub29)

### Steps to reproduce

```
// Uncomment this 2 lines:
// const Test = IocContainer.get("TestClass");
// console.log("Test", Test);
```

### What is expected ?

Worked on the server and in the browser. To avoid errors with https://github.com/inversify/InversifyJS

### What is actually happening?

Error is only on server.

### Additional comments?

This error occurs only on the server, I do not understand why.
This code works:

```
if (process.browser) {
  const Test = IocContainer.get("TestClass");
  console.log("Test", Test);
}
```

This code does not work:

```
if (process.server) {
  const Test = IocContainer.get("TestClass");
  console.log("Test", Test);
}
```

I spent 6 hours on this problem and now it is important for me to solve it as soon as possible. Please give me some hint how to solve this problem.

I have used Inversify.JS many times with Nuxt, but it seems this error has appeared after the 2.4.0+ version, I checked it out.

Help me please. I am sure that the reason for the error in the Nuxt.js.

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c9498">#c9498</a>)</em></sub></div>

**Comment by aldarund** [comment]
Add to your nuxt.config 
```
  render: {
    bundleRenderer: {
      runInNewContext: false
    }
  }
```

**Comment by iliyaZelenko** [comment]
@aldarund you are my savior! I spent 6 hours searching for a problem... **Thank you very much!** 😄🎉  

I found [documentation](https://ssr.vuejs.org/api/#runinnewcontext) about this.

This is not so important, but I just want to clarify: nothing bad will happen to my application if I change this setting? (it is planned to be large)

[bundleRenderer](https://nuxtjs.org/api/configuration-render#bundlerenderer) option: 
> It is recommended to not use this option as Nuxt.js is already providing best SSR defaults and misconfiguration might lead to SSR problems.

Perhaps you see some other best workaround for the error?

Who is interested:  `runInNewContext: 'once'` also gives my error.
